components/tenant-mgt/org.wso2.carbon.tenant.mgt.ui/src/main/resources/web/tenant-mgt/add_tenant.jsp

```diff
@@ -275,7 +275,7 @@ String domainName = request.getParameter("domain");
                             <%if (!isUpdating) {%><span class="required">*</span></td>
                         <%}%>
                         <td colspan="2"><input type="password" name="admin-password"
-                                               id="admin-password" style="width:400px" automcomplete="false"/>
```

Comment:
It should be `autocomplete="false"`

Note: There is an additional 'm' in the original attribute name